File: tests/fixtures/go_repo/src/service/runner.go
package service

import (
	"fmt"
	"example.com/go-fixture/src/domain"
	"example.com/go-fixture/src/helper"
)

type Runner struct {
	helper *helper.Helper
	base   domain.Base
	domain.Embedded
}

func NewRunner() *Runner {
	return &Runner{helper: helper.New()}
}

func (r *Runner) Work(n int) error {
	local := helper.Helper{}
	r.helper.Save(n)
	local.Format()
	f := func(value int) int { return value + 1 }
	_ = f(n)
	fmt.Println(n)
	return nil
}

func (r *Runner) Close() error {
	return nil
}

File: tests/fixtures/csharp_repo/src/Services/Runner.cs
using Demo.Domain;
using HelperAlias = Demo.Services.Helper;

namespace Demo.Services;

public class Runner : BaseEntity, IWorker
{
    private readonly Helper _helper = new Helper();

    public Runner() : base()
    {
    }

    public override void Save()
    {
        Work("save");
        _helper.Format("text");
        Helper local = new Helper();
        local.Format(1);
        Point point = new Point(1, 2);
        Formatter formatter = value => value.Trim();
        formatter("value");
        _ = point.X;
    }

    public void Work(string input)
    {
        Save();
        HelperAlias alias = new HelperAlias();
        alias.Format(input);
    }

    public string Execute(string input)
    {
        return _helper.Format(input);
    }
}
